addons/website_event_track/models/event_track.py

```diff
@@ -145,14 +159,50 @@ def _compute_website_url(self):
 
     # SPEAKER
 
-    @api.depends('partner_id')
-    def _compute_partner_info(self):
+    @api.depends('partner_id', 'partner_id.email')
+    def _compute_contact_email(self):
         for track in self:
             if track.partner_id:
-                track.partner_name = track.partner_id.name
+                track.contact_email = track.partner_id.email
+
+    @api.depends('partner_id', 'partner_id.phone')
+    def _compute_contact_phone(self):
+        for track in self:
+            if track.partner_id:
+                track.contact_phone = track.partner_id.phone
+
+    @api.depends('partner_id')
+    def _compute_partner_email(self):
+        for track in self:
+            if track.partner_id and not track.partner_email:
                 track.partner_email = track.partner_id.email
```

Comment:
Normally yes to give a value. But now that defaults are better computed, maybe it is not necessary anymore.

Here we can simplify some of them like
```
if track.partner_id OR not track.partner_email:
    track.partner_email = track.partner_id.email
```

Using an OR should fix assign compute field possible error.

Reply:
I agree it should help but it does not match the expected behaviour, as the idea is to fill the fields only if nothing is already existing (since contact and speaker info can be different, we do not want to erase existing speaker info when setting a contact partner). Maybe I should indeed add else cases? :)

Reply:
I thought about it : if we want to cover all cases without losing the desired behaviour, which is not to change value if existing already, we could do something like:
`
if track.partner_email:
    pass
else: 
    if track.partner_id:
        track.partner_email = track.partner_id.email
    else:
        track.partner_email = False
`
But that is noisy and a bit weird to have a pass in a compute. What do you think? The same holds for the other ones.

Reply:
There is still the "void / unassigned" issue that @awa-odoo pointed out. Anyway if we have no crash I suppose this is not necessary anymore.